internal/process/vms.go

```diff
@@ -171,6 +171,23 @@ func createVirtualMachineSamples(config *config.Config) {
 			// network
 			if vm.Guest != nil {
 				checkError(config.Logrus, ms.SetMetric("ipAddress", vm.Guest.IpAddress, metric.ATTRIBUTE))
+				var ipAddresses strings.Builder
+				for _, nic := range vm.Guest.Net {
+					// available in api v5
+					if nic.IpConfig != nil {
+						for _, addr := range nic.IpConfig.IpAddress {
+							ipAddresses.WriteString(addr.IpAddress)
```

Comment:
Should we trim trailing '|' from the metric `192.168.0.205|fe80::250:56ff:fea4:175|`. I believe we are currently removing them in other cases

Reply:
done